amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/records/ExportRecordsTests.java

```diff
@@ -0,0 +1,135 @@
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.rm.community.records;
+
+import static java.util.Arrays.asList;
+
+import static org.alfresco.rest.rm.community.model.fileplancomponents.FilePlanComponentType.CONTENT_TYPE;
+import static org.alfresco.rest.rm.community.utils.FilePlanComponentsUtil.createTempFile;
+import static org.alfresco.utility.data.RandomData.getRandomName;
+import static org.alfresco.utility.report.log.Step.STEP;
+import static org.apache.http.HttpStatus.SC_OK;
+
+import org.alfresco.rest.rm.community.base.BaseRMRestTest;
+import org.alfresco.rest.rm.community.model.record.Record;
+import org.alfresco.rest.rm.community.model.recordcategory.RecordCategory;
+import org.alfresco.rest.rm.community.model.recordcategory.RecordCategoryChild;
+import org.alfresco.rest.v0.ExportAPI;
+import org.alfresco.test.AlfrescoTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * This class contains tests for testing the Export functionality on RM site
+ *
+ * @author Shubham Jain
+ * @since 7.1.0
+ */
+public class ExportRecordsTests extends BaseRMRestTest
+{
+    private RecordCategory rootCategory;
+
+    private RecordCategoryChild recordFolder;
+
+    @Autowired
+    private ExportAPI exportAPI;
+
+    @BeforeClass (alwaysRun = true)
+    public void exportRecordsTestsBeforeClass()
+    {
+        STEP("Create root level category");
+        rootCategory = createRootCategory(getRandomName("Category"));
+
+        STEP("Create the record folder inside the rootCategory");
+        recordFolder = createRecordFolder(rootCategory.getId(), getRandomName("Folder"));
+
+    }
+
+    @DataProvider (name = "CreateRMNodes")
+    public Object[][] getRMNodeID()
+    {
+        return new String[][] {
+                { createRecord("Record_4MB", 4).getId() },
+                { createRecord("Record_200MB", 200).getId() },
+                { recordFolder.getId() }
+        };
+    }
+
+    /**
+     * Given a record with size > 4 MB
+     * When I export the record using API
+     * Then the request is successful
+     */
+    @Test (description = "Testing the Export functionality on RM site using API for records of size >4MB and Record " +
+            "Folder containing records with size >4MB",
+            dataProvider = "CreateRMNodes")
+    @AlfrescoTest (jira = "APPS-986")
+    public void exportRMNodeTest(String nodeID)
+    {
+        STEP("Export the created record/record folder with size greater than 4 MB and verifying the expected response" +
+                " code");
+        exportAPI.exportRMNode(getAdminUser().getUsername(), getAdminUser().getPassword(), SC_OK, nodeID);
+    }
+
+    /**
+     * Given few records with collective size > 4 MB
```

Comment:
I would change this to 
Given a list of records with a size > 4MB
When I export the records
Then the request is succesfull